store/rootmulti/store.go

```diff
@@ -1048,7 +1048,7 @@ func (rs *Store) doProofsQuery(req abci.RequestQuery) abci.ResponseQuery {
 	}
 
 	for _, storeInfo := range commitInfo.StoreInfos {
-		res.ProofOps.Ops = append(res.ProofOps.Ops, commitInfo.ProofOp(storeInfo.Name))
+		res.ProofOps.Ops = append(res.ProofOps.Ops, crypto.ProofOp{Key: []byte(storeInfo.Name), Data: storeInfo.CommitId.Hash, })
```

Comment:
Is this same logic that exists in the SDK?

Reply:
No, this was only added here to help debug the app hash problem. Basically, I need a query to see each store's hash at a specified height.

That should help determine which store committed wrong data.